app/views/content/blog/getting-ready-to-apply.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Getting ready to apply - 9 things you can do now
+date: "2021-09-03"
+author: Get Into Teaching
+images:
+  notepad:
+    path: "media/images/content/blog/notepad.jpg"
+    alt: "Someone making notes with a pen on a pad while using a laptop"
+description: |-
+  There are plenty of things you start doing now to lay the groundwork for a successful application. Here's our top nine.
```

Comment:
@christinehitch - I had a go at creating a description. Please feel free to suggest a better one here 😄